base/tomcat-10.1/conf/Catalina/localhost/rewrite.config

```diff
@@ -0,0 +1,21 @@
+# ACME
+RewriteRule ^/acme/login$ /acme/v1/login
+RewriteRule ^/acme/logout$ /acme/v1/logout
+RewriteRule ^/acme/enable$ /acme/v1/enable
+RewriteRule ^/acme/disable$ /acme/v1/disable
+RewriteRule ^/acme/directory$ /acme/v1/directory
+RewriteRule ^/acme/new-nonce$ /acme/v1/new-nonce
+RewriteRule ^/acme/new-account$ /acme/v1/new-account
+RewriteRule ^/acme/new-order$ /acme/v1/new-order
+RewriteRule ^/acme/authz/(.*)$ /acme/v1/authz/$1
+RewriteRule ^/acme/chall/(.*)$ /acme/v1/chall/$1
+RewriteRule ^/acme/order/(.*)$ /acme/v1/order/$1
+RewriteRule ^/acme/acct/(.*)$ /acme/v1/acct/$1
+RewriteRule ^/acme/cert/(.*)$ /acme/v1/cert/$1
+RewriteRule ^/acme/revoke-cert$ /acme/v1/revoke-cert
+
+# EST
+RewriteRule ^/.well-known/est/(.*)$ /est/v1/$1
```

Comment:
These rules are obsolete. Please check the latest `rewrite.config` for Tomcat 9.